tests/basic/upgrade_cluster_test.go

```diff
@@ -74,6 +75,10 @@ var _ = Describe("{UpgradeCluster}", func() {
 
 		printDisks(preUpgradeNodeDisksMap)
 
+		storageNodes := node.GetStorageNodes()
+		pxver, _ := Inst().V.GetDriverVersionOnNode(storageNodes[0])
```

Comment:
add a check to validate len(storageNodes) > 0
 also handle the err in GetDriverVersionOnNode

Reply:
done